src/Components/Components/src/Reflection/ComponentProperties.cs

```diff
@@ -215,19 +214,23 @@ private static void ThrowForCaptureUnmatchedValuesConflict(Type targetType, stri
             throw new InvalidOperationException(
                 $"The property '{parameterName}' on component type '{targetType.FullName}' cannot be set explicitly " +
                 $"when also used to capture unmatched values. Unmatched values:" + Environment.NewLine +
-                string.Join(Environment.NewLine, unmatched.Keys.OrderBy(k => k)));
+                string.Join(Environment.NewLine, unmatched.Keys));
         }
 
         [DoesNotReturn]
         private static void ThrowForMultipleCaptureUnmatchedValuesParameters([DynamicallyAccessedMembers(Component)] Type targetType)
         {
             // We don't care about perf here, we want to report an accurate and useful error.
```

Comment:
This comment is a bit stale now since we're clearly doing stuff in a more perf-conscious manner. Feel free to ignore this if you're otherwise done with this PR, but if you're making more changes it might be less confusing long-term to remove the comment.

Reply:
```suggestion
```